integration_tests/test_issue_332.py

```diff
@@ -0,0 +1,13 @@
+from ltypes import i32
+
+def test_issue_332():
+    arr: i32[4]
+    arr[0] = 1
+    arr[1] = 0
+    arr[2] = -4
+    print(arr[:2])
+    print(arr[1])
+    print(arr[2])
```

Comment:
Use `assert` for verifying that the outputs are actually correct?

Reply:
I am not sure if we can assign Array Section at present (I think only string is supported).
```py
    p: i32[4]
    arr[0] = 1
    arr[1] = 0
    arr[2] = -4
    p = arr[:2]
```
Throws this:
```
LFORTRAN_ASSERT(ASR::is_a<ASR::Character_t>(*ASRUtils::expr_type(x.m_v)) &&
AssertFailed: ASR::is_a<ASR::Character_t>(*ASRUtils::expr_type(x.m_v)) && n_dims == 0
```